modules/hostap/src/supp_events.c

```diff
@@ -335,7 +335,11 @@ int supplicant_send_wifi_mgmt_ap_sta_event(void *ctx,
 
 	if (event == NET_EVENT_WIFI_CMD_AP_STA_CONNECTED) {
 		sta_info.link_mode = get_sta_link_mode(ap_ctx, sta);
+#ifdef CONFIG_IEEE80211AX
```

Comment:
why not move this check inside `is_twt_capable`?

Reply:
Yes, move to `is_twt_capable`.